testsuite/features/build_validation/reposync/srv_sync_all_products.feature

```diff
@@ -373,6 +373,46 @@ Feature: Synchronize products in the products page of the Setup Wizard
     When I use spacewalk-common-channel to add channel "sle-micro-5.5-devel-uyuni-client" with arch "x86_64"
     And I wait until the channel "sle-micro-5.5-devel-uyuni-client-x86_64" has been synced
 
+@susemanager
+@slmicro60_minion
+  Scenario: Add SUSE Linux Micro 6.0
+    Given I am authorized for the "Admin" section
+    When I follow the left menu "Admin > Setup Wizard > Products"
+    And I wait until I do not see "currently running" text
+    And I wait until I do not see "Loading" text
+    And I enter "SUSE Linux Micro 6.0" as the filtered product description
+    And I select "SUSE Linux Micro 6.0 x86_64" as a product
+    Then I should see the "SUSE Linux Micro 6.0 x86_64" selected
+    When I open the sub-list of the product "SUSE Linux Micro 6.0 x86_64"
+    And I select "SUSE Manager Client Tools for SL Micro 6 x86_64" as a product
+    Then I should see the "SUSE Manager Client Tools for SL Micro 6 x86_64" selected
```

Comment:
```suggestion
    And I select "SUSE Manager Client Tools for SUSE Linux Micro 6 x86_64" as a product
    Then I should see the "SUSE Manager Client Tools for SUSE Linux Micro 6 x86_64" selected
```
I hope this partial match is sufficient.

Reply:
checked and corrected, will test as soon as the port is also adjusted